ztp/siteconfig-generator-kustomize-plugin/Makefile

```diff
@@ -8,6 +8,7 @@ KUSTOMIZE_VERSION=4.4.0
 KUSTOMIZE := $(if $(shell command -v kustomize 2>/dev/null), $(shell command -v kustomize), $(KUSTOMIZE_BIN))
 SITECONFIG_DIR := ../siteconfig-generator
 EXTRAMANIFEST_DIR := ../source-crs/extra-manifest
+TEST_DIR = ./testManifest
```

Comment:
This change will be reverted when unit test is added in next commit.

Reply:
strange comment :) , just make the required tests in the same PR

Reply:
Deleted with commit 4e71357842c233ccb388973c901535f4c82919cd